Jenkinsfile

```diff
@@ -71,26 +71,50 @@ pipeline {
       parallel {
         /*
          * A scan of the conjur image is skipped since this image is scanned
-         * vulnerabilities for builds in the cyberark/conjur repository.
+         * vulnerabilities for builds in the cyberark/conjur repository and we
+         * make no changes to the container here.
+         *
+         * `false` in the 3rd parameter to scanAndReport means to ignore issues
+         * with no fix. `true` means to include those issues in the report.
          */
+        stage('Scan deployer image for fixable vulns') {
+          steps {
+            scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:${TAG}", "HIGH", false)
```

Comment:
Could both of these scans be stacked together in each step for each image? e.g.:
```
scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:${TAG}", "NONE", true)
scanAndReport("gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:${TAG}", "HIGH", false)
```

Reply:
If I stack them in a single step, won't they run sequentially rather than in parallel?

Reply:
yeah that's a good point. ignore this but do the change for the grouping by image (vs level)